java/src/client/poller/Poller.java

```diff
@@ -21,9 +24,22 @@
 	 * 
 	 * @param server interface
 	 */
-	public Poller(IServer server)
+	public Poller()
 	{
-		this.server = server;
+		this.clientModel = new ClientModel();
+		this.timer = new Timer();
+		timer.scheduleAtFixedRate(new TimerTask(){
+			@Override
+			public void run(){
+				GameModel_Input input = new GameModel_Input();
+				input.setVersion(clientModel.getVersion());
+				GameModel_Output output = SessionManager.get_instance().getServer().getModel(input);
+				if (!output.getResponse().equals("true")) {
+					//this.clientModel = SessionManager.get_instance().getInterpreter().deserialize(output.getResponse());
```

Comment:
This previous line with the commented out portion is needed and was waiting on Trevor's portion. That's exactly how we predict the method being called on his object